tests/cpp/test_argsort.cpp

```diff
@@ -126,13 +127,14 @@ TEST_F(ArgsortTest, Predication) {
   auto tv0 = makeContigConcreteTensor(shape);
   fusion.addInput(tv0);
 
-  auto tv1 = argsort(tv0, -1);
-  auto tv2 = set(tv1);
-  fusion.addOutput(tv2);
+  auto tv1 = set(tv0);
```

Comment:
`set` is added here because the new validation mandates the input is a Local tensor. That can be relaxed but makes the lowering much simpler.